pkg/internal/containerd_conf.toml

```diff
@@ -68,12 +76,16 @@ version = 2
       bin_dir = "C:\\k\\cni"
       conf_dir = "C:\\k\\cni\\config"
       conf_template = ""
+      ip_pref = "ipv4"
```

Comment:
was this set manually to ipv4? or is this also generated

Reply:
It was set to `""`. `""` and `ipv4` are [synonymous](https://github.com/containerd/containerd/blob/992067339161f7e649a23db160943c8b4709c790/docs/cri/config.md?plain=1#L412). I modified it to `ipv4` to make it clear.